src/lib/airtable/request.js

```diff
@@ -646,6 +665,31 @@ export const updateManyInventorys = async (recordUpdates) => {
   return Promise.all(updatePromises);
 };
 
+// NONGENERATED: Review purchase request
+export const reviewPurchaseRequest = async (purchaseRequest) => {
```

Comment:
TBH don't know if i really like the name `reviewPurchaseRequest`. Maybe `updatePurchaseRequestAndInventory`? I think my main concern is just that "review" doesn't seem like an action that should persist something.